jetbrains-core/resources/telemetryOverride.json

```diff
@@ -507,6 +512,24 @@
                     "required": false
                 }
             ]
+        },
+        {
+            "name": "codeTransform_projectContains1pDependencies",
```

Comment:
new metrics should be declared in https://github.com/aws/aws-toolkit-common

Reply:
PR to add this metric in toolkit-common 
https://github.com/aws/aws-toolkit-common/pull/683